dub.sdl

```diff
@@ -31,6 +31,8 @@ subPackage {
     "src/dmd/utf.d" \
     "src/dmd/utils.d"
 
+  versions "callback_API"
```

Comment:
I think this version should follow the usual style, `CallbackAPI` (preferred) or `Callback_API`